chromium_src/content/browser/renderer_host/clipboard_host_impl.cc

```diff
@@ -24,8 +24,12 @@ std::u16string Sanitize(content::ContentBrowserClient* client,
 
   const GURL url(data);
   if (url.is_valid() && !url.is_empty()) {
```

Comment:
Let's make sure we only ever modify HTTP/HTTPS URLs:
```suggestion
  if (url.is_valid() && !url.is_empty() && url.SchemeIsHTTPOrHTTPS()) {
```